e2e/react-router/basic-file-based-code-splitting/src/routes/_layout/_layout-2/layout-a.tsx

```
@@ -1,4 +1,5 @@
-export const Route = createFileRoute({
+import { createFileRoute } from '@tanstack/react-router'
```

ESLint

[error] Expected 1 empty line after import statement not followed by another import.

(import/newline-after-import)